src/content/structured/components/page-header/code.mdx

```diff
@@ -480,3 +481,70 @@ export const withReactRouter = [
     <IcSectionContainer />
   </MemoryRouter>
 </ComponentPreview>
+
+### With Slots
+
+export const withSlots = [
+  {
+    language: "Web component",
+    snippet: `<ic-page-header>
+      <ic-typography variant="h1" slot="heading">Latte recipe</ic-typography>
+      <ic-typography slot="subheading">A Latte is a popular Italian coffee, made with espresso, steamed milk and a thin layer of foam.</ic-typography>
+      <ic-chip slot="heading-adornment" label="BETA" size="large"></ic-chip>
+      <ic-breadcrumb-group slot="breadcrumbs">
+        <ic-breadcrumb current="true" page-title="Drinks" href="#"></ic-breadcrumb>
+        <ic-breadcrumb page-title="Coffees" href="#"></ic-breadcrumb>
+        <ic-breadcrumb page-title="Latte" href="#"></ic-breadcrumb>
+      </ic-breadcrumb-group>
+      <ic-text-field slot="input" placeholder="Search for ingredients…" label="Input" hide-label="true" />
+</ic-page-header>`,
```

Comment:
So the indentation is still not quite right. I think line 491-499 needs to be outdented to remove the empty spaces.

Reply:
Same with the React snippet:
![image](https://github.com/mi6/ic-design-system/assets/116016939/02f13dbc-5c74-47ef-b556-0ecf53572bc0)